buildSrc/src/main/java/Dependencies.kt

```diff
@@ -176,7 +182,13 @@ object Dependencies {
     }
 
     object Calling {
-        const val avs =  "com.wire:avs:${Versions.avs}"
+        const val avs = "com.wire:avs:${Versions.avs}"
         const val jna = "net.java.dev.jna:jna:${Versions.jna}"
     }
+
+    object Firebase {
+        const val firebaseBOM = "com.google.firebase:firebase-bom:${Versions.firebaseBOM}"
+        const val firebaseCloudMessaging = "com.google.firebase:firebase-messaging-ktx"
+        const val abcNotifications = "com.linecorp.abc:kmm-notifications:${Versions.abcNotifications}"
+    }
```

Comment:
Same here.

I think `Firebase` is not a concept we should have in Kalium.

Apps may use Firebase or not. Kalium just receives tokens (be them from Firebase or not) and pushes them to the backend.